### DISCUSSION TITLE
Can QStash work with 3rd party APIs

### DISCUSSION THREAD

**Discussion by taskclone** [upstash/product-roadmap#35] [Announcements]
Seems your model is to glue together microservices we control.  Is it possible to ingest webhooks that we would direct your way, but where it's actually coming from a third party API?  We don't control the payload or headers in that instance.  

I know we could first send it to a function to add the extra stuff and then to you, but that seems redundant.

**Comment by sancar** [comment]
@taskclone Your question is moved here https://github.com/upstash/qstash-js/issues/251
